spaces/aliases.json

```
@@ -64,5 +64,6 @@
   "goodghosting.eth": "halofi.eth",
   "arbitrum-odyssey.eth": "arbitrumfoundation.eth",
   "pokt-network": "poktdao.eth",
-  "lensterapp.eth": "lenster.xyz"
+  "lensterapp.eth": "lenster.xyz",
+  "nexondao.eth": "eralend.eth"
```

Create a proposal on the source space as well as on the target space with the title "Migrate this space" from the respective controllers of the spaces (the Ethereum address that is registered in the snapshot ENS text record.
https://docs.snapshot.org/user-guides/spaces/migrate-or-delete#migrate-a-space
```suggestion
  "nexondao.eth": "eralend.eth"
```